_includes/store/liaraalis-small-business-rule-bundle.md

```diff
@@ -0,0 +1,5 @@
+This plugin provides the possibility to use the small business regulation, as it can be applied in Germany and Austria, when creating invoices in Kimai 2.
```

Comment:
```suggestion
This plugin provides the ability to use the small business regulation, as it can be applied in Germany and Austria, when creating invoices in Kimai.
```

Reply:
Not sure about the "ability" va "possibility"!